flink-addons/flink-language-binding/src/main/python/org/apache/flink/languagebinding/api/python/dill/__diff.py

```diff
@@ -0,0 +1,247 @@
+################################################################################
```

Comment:
No, you can not.

I believe Spark had special license deal, as part of AmpLab before, to
allow them to get license friendly to Apache license before.

- Henry

On Mon, Feb 2, 2015 at 11:52 AM, Robert Metzger <notifications@github.com>
wrote:

> In
> flink-addons/flink-language-binding/src/main/python/org/apache/flink/languagebinding/api/python/dill/__diff.py
> <https://github.com/apache/flink/pull/202#discussion_r23952693>:
>
> > @@ -0,0 +1,247 @@
> > +################################################################################
>
> We need to find another solution here with the licenses. I think we can
> not just re-distribute this file with our license.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/flink/pull/202/files#r23952693>.
>